eversql/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bc900600-d0cf-4ddf-83b7-cdaba44d1525",
+  "app_id": "eversql",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Monitoring",
+    "Category::Automation",
+    "Category::Languages"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Automatic SQL and Database Tuning for MySQL, PostgreSQL, Aurora",
+    "title": "EverSQL- SQL query optimization, powered by AI",
+  "media": [   
+      {
+        "caption": "EverSQL SQL optimization",
+        "image_url": "images/1.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "EverSQL query diff",
+        "image_url": "images/2.png",
+        "media_type": "image"
+      },
+{
+        "caption": "EverSQL supported databases",
+        "image_url": "images/3.png",
+        "media_type": "image"
+      },
+{
+        "caption": "EverSQL supported OS",
+        "image_url": "images/4.png",
+        "media_type": "image"
+      }
+
+	]
+
+  },
+  "author": {
+    "homepage": "https://eversql.com",
+    "name": "EverSQL",
+    "support_email": "support@eversql.com"
```

Comment:
The author object is actually missing a required filed - sales email. Can you fill it out using the syntax below?

"sales_email": "xyz@xyz.com"